packages/gatsby-remark-images/src/__tests__/gatsby-node.js

```diff
@@ -102,4 +103,210 @@ describe(`pluginOptionsSchema`, () => {
       expect(isValid).toBe(false)
     })
   })
+
+  describe(`allow different variants of "tracedSVG" option`, () => {
+    describe(`supports boolean variant`, () => {
+      it.each([
+        [`true`, true],
+        [`false`, false],
+      ])(`%s`, async (_title, booleanValue) => {
+        const { isValid } = await testPluginOptionsSchema(pluginOptionsSchema, {
+          tracedSVG: booleanValue,
+        })
+
+        expect(isValid).toBe(true)
+      })
+    })
+
+    describe(`supports object notation`, () => {
+      it(`should validate when all fields are set`, async () => {
+        const { isValid } = await testPluginOptionsSchema(pluginOptionsSchema, {
+          tracedSVG: {
+            turnPolicy: Potrace.TURNPOLICY_RIGHT,
+            turdSize: 50,
+            alphaMax: 0.5,
+            optCurve: false,
+            optTolerance: 0.9,
+            threshold: 230,
+            blackOnWhite: false,
+            color: `red`,
+            background: `green`,
+          },
+        })
+
+        expect(isValid).toBe(true)
+      })
+
+      it(`should validate when some fields are set`, async () => {
+        const { isValid } = await testPluginOptionsSchema(pluginOptionsSchema, {
+          tracedSVG: {
+            turnPolicy: Potrace.TURNPOLICY_RIGHT,
+            turdSize: 50,
+            // alphaMax: 0.5,
+            // optCurve: 0.2,
+            // optTolerance: 0.9,
+            // threshold: 230,
+            // blackOnWhite: false,
+            color: `red`,
+            background: `green`,
+          },
+        })
+
+        expect(isValid).toBe(true)
+      })
+
+      it(`should fail validation when unknown fields are set`, async () => {
+        const { isValid, errors } = await testPluginOptionsSchema(
+          pluginOptionsSchema,
+          {
+            tracedSVG: {
+              foo: `bar`,
+            },
+          }
+        )
+
+        expect(isValid).toBe(false)
+        expect(errors).toMatchInlineSnapshot(`
+          Array [
+            "\\"tracedSVG.foo\\" is not allowed",
+          ]
+        `)
+      })
+
+      describe(`turnPolicy variants`, () => {
+        it.each([
+          `TURNPOLICY_BLACK`,
+          `TURNPOLICY_WHITE`,
+          `TURNPOLICY_LEFT`,
+          `TURNPOLICY_RIGHT`,
+          `TURNPOLICY_MINORITY`,
+          `TURNPOLICY_MAJORITY`,
+        ])(`supports "%s" by policy name`, async name => {
+          const { isValid } = await testPluginOptionsSchema(
+            pluginOptionsSchema,
+            {
+              tracedSVG: { turnPolicy: Potrace[name] },
```

Comment:
Ops, this is actually wrong - I need to edit this

Reply:
Just for clarification - this test should pass. The wrong part is that this `it.each` and following it.each` were both passing value (and it should pass policy name in one block and value in another)

Reply:
was addressed https://github.com/gatsbyjs/gatsby/pull/28242/commits/6990d080bb4f5f17396ab0adcfcd8ad4538f401f